{{cookiecutter.project_slug}}/.github/workflows/tests.yml

```diff
@@ -0,0 +1,47 @@
+name: Tests
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    paths-ignore:
+      - "**.md"
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version:
+        {%- for python_version in range(
+          cookiecutter.min_python_version | replace('3.', '') | int,
+          cookiecutter.max_python_version | replace('3.', '') | int
+        ) %}
+          - "3.{{ python_version }}"
+        {%- endfor %}
```

Comment:
I could get this syntax to work, but not the `[]` one. Missed this first time around.